backend/Tasque.BLL/Services/ProjectService.cs

```diff
@@ -29,15 +29,16 @@ public List<ProjectDto> GetProjectsByOrganizationId(int organizationId)
         return _mapper.Map<List<ProjectDto>>(_db.Projects.Where(p => p.OrganizationId == organizationId));
     }
 
-    public ProjectDto GetProjectById(int id)
+    public async Task<ProjectDto> GetProjectById(int id)
     {
-        return _mapper.Map<ProjectDto>(_db.Projects
+        var project = await _db.Projects
```

Comment:
Maybe it's better to use `.Single` instead of `.FirstOfDefault` here ? Null-related exceptions will be automatically handled on the server side then.